crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -481,6 +487,32 @@ def services
           return
         end
 
+        begin
+          unless monasca_node.nil? || database_node.nil?
+            database_node.wait_for_script_to_finish(
+              "/usr/sbin/crowbar-dump-monasca-db.sh",
+              timeouts[:dump_shutdown_monasca_db]
```

Comment:
the timeout name is misleading now after you moved the shutdown code out of the dump script.

Reply:
Indeed...fixed.